examples/caller/make_multiple_calls.py

```diff
@@ -0,0 +1,53 @@
+import threading
+import uuid
+from threading import Thread
+from typing import List
+
+from ondewo.vtsi.client import VtsiClient
+from ondewo.vtsi.voip_pb2 import StartCallInstanceResponse
+
+#     SIP
+SIP_SIM_VERSION: str = "1.5.4"
+
+#     VTSI_SERVER
+VTSI_HOST: str = "0.0.0.0"
```

Comment:
delete all this, you dont need it-- the defaults will use grpc-vtsi.ondewo.com. so the only thing you need is the grpc certificate. make a note of that.

Reply:
get_minimal_client() requires VTSI_HOST and VTSI_PORT, I can delete them only in the case of full client config